docs(interlink): add peer asn and vlan id for L2 and self hosted (#6807)
scaleway/docs-content: pages/interlink/how-to/order-interlink.mdx

   <Tabs>
    
        <TabsTab label="Hosted">

>         1. Select a [partner](/interlink/concepts/#partner). To establish an InterLink connection via a partner, generally you must be a client of that partner, e.g. with your infrastructure in their datacenter, or on your own premises using the partner as a network provider. [Get help with choosing a partner](/interlink/reference-content/overview/#hosted-interlinks).

Reviewer: ```suggestion
        1. Select a [partner](/interlink/concepts/#partner). To establish an InterLink connection via a partner, you must be a client of that partner with, for example, your infrastructure in their datacenter or on your own premises using the partner as a network provider. [Get help with choosing a partner](/interlink/reference-content/overview/#hosted-interlinks).
```

Je tente de fluidifier / simplifier la phrase avec cette proposition, mais il y a des chances que Firdevs ou Vanda s'en sortent mieux que moi ^^ N'hésitez pas à challenger !

Reviewer 2: Here's my version:

To establish an InterLink connection via a partner, you must be a client of that partner, e.g., by hosting your infrastructure in their datacenter, or by using the partner as your network provider on‑premises. [Get help with choosing a partner](/interlink/reference-content/overview/#hosted-interlinks).

Reviewer 3: Encore un ajustement :) 

1. Select a [partner](/interlink/concepts/#partner). To establish an InterLink connection through a partner, you must be a client of that partner — for example, by hosting your infrastructure in their data center, or by using the partner as your network provider on-premises. See [How to choose a partner](https://github.com/interlink/reference-content/overview/#hosted-interlinks) for help.

>         2. Select the [location](/interlink/concepts/#location) for your InterLink. This represents the physical location where the connection to Scaleway's network will be established.

Reviewer: - Peut-être quelque chose de plus simple que "represents" ? "This is the location where" ? 
- New guidelines : Il faut tourner la deuxième partie de la phrase autrement pour éliminer le possessif 's por favor

>         2. Select the [location](/interlink/concepts/#location) for your InterLink. This represents the physical location where the connection to Scaleway's network will be established.

Reviewer: ```suggestion
        2. Select the [location](/interlink/concepts/#location) for your InterLink. This is the physical location where the connection to the Scaleway network will be established.
```

Peut-être quelque chose de plus simple que "represents" ? "This is the location where" ?
New guidelines : Il faut tourner la deuxième partie de la phrase autrement pour éliminer le possessif 's por favor

>         2. Select the [location](/interlink/concepts/#location) for your InterLink. This represents the physical location where the connection to Scaleway's network will be established.

Reviewer 3: ```suggestion
        2. Select the [location](/interlink/concepts/#location) for your InterLink. This is the physical location where the connection to the Scaleway network will be established.
```

>         3. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This represents the maximum bandwidth that will be available over the InterLink.

Reviewer: ```suggestion
        3. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This is the maximum bandwidth available over the InterLink.
```

Idem pour represents, feel free to choose another way to simplify if too much meaning is lost ^^

Reviewer 2: Choose a bandwidth level?
(Without fully understanding the context, maybe "level" sounds better.)

>         3. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This represents the maximum bandwidth that will be available over the InterLink.

Reviewer 3: ```suggestion
        3. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This is the maximum bandwidth available over the InterLink.
```

>         4. If you selected a partner offering L2 connectivity, enter your peer ASN, and optionally, a VLAN ID.

Reviewer: ```suggestion
        4. If you selected a partner offering L2 connectivity, enter your peer ASN and, optionally, a VLAN ID.
```

Purement stylistique, je trouve l'incise plus fluide

        </TabsTab>
        <TabsTab label="Self-hosted">

>         1. Select a [dedicated connection](/interlink/concepts/#dedicated-physical-connection). The InterLink will be hosted over this connection. You must have already provisioned this connection via the Scaleway sales team. [Contact sales](https://www.scaleway.com/en/contact-sales/) if you would like to provision a dedicated connection: this cannot be done via the Scaleway console or API.

Reviewer 2: ```suggestion
        1. Select a [dedicated connection](/interlink/concepts/#dedicated-physical-connection). The InterLink will be hosted over this connection. You must have already provisioned this connection via the Scaleway Sales team. [Contact Sales](https://www.scaleway.com/en/contact-sales/) if you would like to provision a dedicated connection: this cannot be done via the Scaleway console or API.
```

Reviewer 2: According to our new guidelines, we will be spelling "Support" with an uppercase "S", so I guess we could apply the same rule to "Sales" too.

>         1. Select a [dedicated connection](/interlink/concepts/#dedicated-physical-connection). The InterLink will be hosted over this connection. You must have already provisioned this connection via the Scaleway sales team. [Contact sales](https://www.scaleway.com/en/contact-sales/) if you would like to provision a dedicated connection: this cannot be done via the Scaleway console or API.

Reviewer 3: ```suggestion
        1. Select a [dedicated connection](/interlink/concepts/#dedicated-physical-connection). The InterLink is hosted over this connection. You must have already provisioned this connection through the Scaleway sales team — this cannot be done via the Scaleway console or API. [Contact sales](https://www.scaleway.com/en/contact-sales/) to provision a dedicated connection.
```

>         2. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This represents the maximum bandwidth that will be available over the InterLink.

Reviewer: ```suggestion
        2. Choose a [bandwidth](/interlink/concepts/#bandwidth) amount. This is the maximum bandwidth available over the InterLink.
```

Reviewer 2: Bandwidth level (instead of amount)?

>         3. Enter your peer ASN, and optionally, a VLAN ID.

Reviewer: ```suggestion
        3. Enter your peer ASN and, optionally, a VLAN ID.
```
